src/api/status/public/js/post/index.js

```diff
@@ -0,0 +1,19 @@
+const totalPostsEl = document.getElementById('totalPosts');
```

Comment:
This is dangerous to do outside of onload, since the DOM may or may not be ready, and you'll get race conditions.

Reply:
Move it to line 11, when you use it.